src/components/Reports/StaffExpenseReport/DownloadButtonGroup/downloadReport.tsx

```diff
@@ -0,0 +1,74 @@
+import { DateTime } from 'luxon';
+import { Transaction } from 'src/components/Reports/StaffExpenseReport/StaffExpenseReport';
+import { ReportType } from '../Helpers/StaffReportEnum';
+
+const csvHeader = ['Date', 'Category', 'Amount'];
+
+const makeTable = (
+  title: string,
+  transactions: Transaction[],
+  type: ReportType,
+) => {
+  if (type === ReportType.Income) {
+    transactions = transactions.filter((transaction) => transaction.total > 0);
+  } else if (type === ReportType.Expense) {
+    transactions = transactions.filter((transaction) => transaction.total < 0);
+  }
+  const rows = transactions.map((transaction) =>
+    [transaction.month, transaction.category, transaction.total].join(','),
+  );
+  return [title, csvHeader.join(','), ...rows, ''].join('\n');
+};
+
+export const downloadCsv = (
+  type: ReportType,
+  enqueueSnackbar: (message: string, options?: { variant: string }) => void,
+  transactions: Transaction[] | undefined,
+) => {
+  if (!Object.values(ReportType).includes(type)) {
```

Comment:
The type of `type` already ensures that this is a `ReportType`. I would remove this check or change the type to `string` if it can actually be an arbitrary value.